src/librustc/ty/maps.rs

```diff
@@ -599,7 +611,8 @@ define_maps! { <'tcx>
 
     [] describe_def: DescribeDef(DefId) -> Option<Def>,
     [] def_span: DefSpan(DefId) -> Span,
-
+    [] stability: Stability(DefId) -> Option<attr::Stability>,
+    [] deprecation: Deprecation(DefId) -> Option<attr::Deprecation>
```

Comment:
@nikomatsakis how do you prefer this to be implemented. I noticed that @cramertj and I diverged here. @cramertj is using the `metadata_dep_node` function while I am declaring new `DepNode` methods for these, as suggested by @eddyb in #41593.

I just want alignment here since we still have majority of the queries to convert.